src/lib/orchestrator.ts

```diff
@@ -201,6 +203,30 @@ export async function getMetricImplementation(id: string): Promise<MetricImpleme
         });
 }
 
+/**
+ * Retrieves a particular metric configuration from the orchestrator service.
+ * 
+ * @param cloud_service_id the Cloud Service ID
+ * @param metric_id the metric id
+ * @returns metric configuration
+ */
+export async function getMetricConfiguration(cloud_service_id: string, metric_id: string): Promise<MetricConfiguration> {
```

Comment:
minor: proper camel casing: `cloudServiceId` (or I think we just used `serviceId` mostly) and `metricId`.